### PR TITLE
New package: DimitrovAdrian.RcloneTray version 1.0.0

### PR DESCRIPTION
**Pull request by jo-chemla**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/190847)

### CODE REVIEW

**Comment by jo-chemla** [comment]
Not sure what silent tag I should try and how to test them quickly, would love some feedback from the winget team. Thanks a lot

**Comment by jo-chemla** [comment]
The manifest installation does work correctly, but indeed require a single user input because of the below Defender protection. Is there a way to handle unknown publishers via eg a winget-cli flag? 

`Microsoft Defender SmartScreen prevented unrecognized app from starting`

image

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 897.)

File: manifests/d/DimitrovAdrian/RcloneTray/1.0.0/DimitrovAdrian.RcloneTray.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+ReleaseDate: 2021-12-07
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/d/DimitrovAdrian/RcloneTray/1.0.0/DimitrovAdrian.RcloneTray.installer.yaml:17]:
```suggestion
- Architecture: x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> Automatic Validation ended with: 
Edit: Sorry for the delay - my automation doesn't have a way to post these images into their PR, so they sit open on my laptop, until I can match them.
> 
> image

**Comment by stephengillie** [comment]
Hi @jo-chemla

> The manifest installation does work correctly, but indeed require a single user input because of the below Defender protection.

In the automated pipeline, this is disabled or automated in some way. I'm not sure of the specific implementation, but this warning isn't a blocker. For my manual validation VMs, I click through this, and it's also not counted as a blocker. 

> * Is there a way to handle unknown publishers via eg a winget-cli flag?

I'm not sure that I understand the question. Publisher data should be in manifests, and I do not know of a way to enter it directly into the package manager application. Which application has an unknown publisher - and unknown to whom?  

> * Also, I submitted the exe as a false positive [here](https://learn.microsoft.com/en-us/defender-endpoint/defender-endpoint-false-positives-negatives#submit-a-file-to-microsoft-for-analysis) as advised in the [Troubleshoot section](https://github.com/microsoft/winget-pkgs/blob/aa7b75bf26d86c12135afebbdb1b3211b1351c14/doc/Troubleshoot.md#binary-validation-error)

I hadn't connected these ideas before, but it seems like there are a couple of levels of Smartscreen. The upper level blocks similar to a Defender failure. But this seems like a lower level, where a warning is given to users, which can be bypassed.

**Comment by stephengillie** [comment]
Verify with architecture change:
image
image

**Comment by jo-chemla** [comment]
Thanks for the feedback. My comments were mostly trying to address the non-validation by the azure pipeline in case Defender SmartScreen was preventing adding this package to the winget-pkgs repository - when I was mentioning `Unknown Publisher`, this was in the Defender SmartScreen dialog, where I wanted to know if a winget-flag could bypass and let the user allow installation by unknown publishers in the smart-screen sense. 

Thanks for the last bits as well - finally I wasn't able to submit the package as a false positive on the link provided in the doc. 

+I just made the PR to modify the architecture to x64

**Review by stephengillie — APPROVED** [review]

**Pull request merged**